cms/templates/js/course-outline.underscore

```diff
@@ -44,7 +44,8 @@ if (xblockInfo.get('graded')) {
     <div class="<%= xblockType %>-header">
         <% if (includesChildren) { %>
             <h3 class="<%= xblockType %>-header-details expand-collapse <%= isCollapsed ? 'expand' : 'collapse' %> ui-toggle-expansion"
-                title="<%= interpolate(gettext('Collapse/Expand this %(xblock_type)s'), { xblock_type: xblockTypeDisplayName }, true) %>">
+                title="<%= interpolate(
+				gettext('Collapse/Expand this %(xblock_type)s'), { xblock_type: xblockTypeDisplayName }, true) %>">
```

Comment:
Also please make this style change througout the file. Ping me when you want me to take another look and I will test it out.